app/assets/stylesheets/pdf.css

```diff
@@ -65,7 +65,7 @@ body {
 .field,
 .declaration {
   margin-bottom: 10px;
-  color: rgb(51, 51, 51);
+  color: #333333;
```

Comment:
Oh sorry, I did comment on both, but I meant to rollback all of your changes to pdf. 🙂

Reply:
Oops, sorry. Fixing it now.